osu.Server.Spectator/Hubs/Multiplayer/ServerMultiplayerRoom.cs

```diff
@@ -23,24 +23,28 @@ public MatchTypeImplementation MatchTypeImplementation
             private set => matchTypeImplementation = value;
         }
 
-        public readonly MultiplayerQueue Queue;
+        public IPlaylistImplementation PlaylistImplementation => MatchTypeImplementation.Playlist;
```

Comment:
Can we call this just `Playlist`? What is the meaning behind "Implementation" here?

Reply:
`Playlist` would hide the base class' member, and I don't know the best way to do this:

https://github.com/ppy/osu-server-spectator/blob/18bd8d164d364bb90fdb6b10328b0168218f64f3/osu.Server.Spectator.Tests/Multiplayer/MatchTypeRoomEventHookTests.cs#L22-L32

Reply:
It would be nice is `ServerMultiplayerRoom` could just implement `IPlaylistImplementation` and delegate to the implementation internally. But having `Initialize` in that interface feels really bad..

Reply:
I've gone ahead and implemented the idea in https://github.com/ppy/osu-server-spectator/pull/301#discussion_r2300265527 in the last two commits. Please see if you agree with this structure more.